backend-api/template.yaml

```diff
@@ -1158,18 +1158,9 @@ Resources:
 Outputs:
   SessionsApiUrl:
```

Comment:
I've approved this PR but I thought just now that it might be nice to export this value so that I can reference it in the `sts-mock` template like this `ASYNC_BACKEND_BASE_URL: !ImportValue mob-async-backend-SessionsApiUrl`

What do you think? 🤔 I think this should work...

```
    Export:
      Name:
        Fn::Sub: ${AWS::StackName}-SessionsApiUrl
```

Reply:
Left a similar comment on your PR - we need to avoid tight coupling of stacks otherwise you enforce the deployment order of the stacks (i.e. deploy the sts-mock stack first and then deploy the backend-api stack)

Reply:
Will avoid for now - unless we find it becomes needed :)

Reply:
Sounds good to me :)